src/code/graph.c

```diff
@@ -45,23 +45,28 @@ void UCodeDisas_Disassemble(UCodeDisas*, Gfx*);
 void Graph_DisassembleUCode(Gfx* workBuf) {
     UCodeDisas disassembler;
 
-    if (HREG(80) == 7 && HREG(81) != 0) {
+    if (R_HREG_MODE == HREG_MODE_UCODE_DISAS && R_UCODE_DISAS_TOGGLE != 0) {
         UCodeDisas_Init(&disassembler);
-        disassembler.enableLog = HREG(83);
+        disassembler.enableLog = R_UCODE_DISAS_LOG_LEVEL;
+
         UCodeDisas_RegisterUCode(&disassembler, ARRAY_COUNT(D_8012D230), D_8012D230);
         UCodeDisas_SetCurUCode(&disassembler, gspF3DZEX2_NoN_PosLight_fifoTextStart);
+
         UCodeDisas_Disassemble(&disassembler, workBuf);
-        HREG(93) = disassembler.dlCnt;
-        HREG(84) = disassembler.tri2Cnt * 2 + disassembler.tri1Cnt + (disassembler.quadCnt * 2) + disassembler.lineCnt;
-        HREG(85) = disassembler.vtxCnt;
-        HREG(86) = disassembler.spvtxCnt;
-        HREG(87) = disassembler.tri1Cnt;
-        HREG(88) = disassembler.tri2Cnt;
-        HREG(89) = disassembler.quadCnt;
-        HREG(90) = disassembler.lineCnt;
-        HREG(91) = disassembler.syncErr;
-        HREG(92) = disassembler.loaducodeCnt;
-        if (HREG(82) == 1 || HREG(82) == 2) {
+
+        R_UCODE_DISAS_DL_COUNT = disassembler.dlCnt;
+        R_UCODE_DISAS_TOTAL_COUNT =
+            disassembler.tri2Cnt * 2 + disassembler.tri1Cnt + (disassembler.quadCnt * 2) + disassembler.lineCnt;
+        R_UCODE_DISAS_VTX_COUNT = disassembler.vtxCnt;
+        R_UCODE_DISAS_SPVTX_COUNT = disassembler.spvtxCnt;
+        R_UCODE_DISAS_TRI1_COUNT = disassembler.tri1Cnt;
+        R_UCODE_DISAS_TRI2_COUNT = disassembler.tri2Cnt;
+        R_UCODE_DISAS_QUAD_COUNT = disassembler.quadCnt;
+        R_UCODE_DISAS_LINE_COUNT = disassembler.lineCnt;
+        R_UCODE_DISAS_SYNC_ERROR = disassembler.syncErr;
```

Comment:
Since this is an amount of sync errors
```suggestion
        R_UCODE_DISAS_SYNC_ERRORS = disassembler.syncErr;
```